docs/doctest.sh

```
@@ -44,6 +38,20 @@ if ! [ "${YASH+set}" ]; then
     export YASH
 fi
 
+if [ $# -eq 0 ]; then
+    # scan all files
+
+    # prepare for parallel execution
+    nproc="$({ nproc || sysctl -n hw.logicalcpu; } 2>/dev/null || echo 1)"
+    # if xargs supports -P, use it
+    if [ "$nproc" -gt 1 ] && echo true | xargs -L 1 -P "$nproc" sh -c 2>/dev/null; then
+        find "$(dirname -- "$0")" -type f -name '*.md' | xargs -L 1 -P "$nproc" "$0"
```

Shellcheck (0.10.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)